src/timepicker-select/timepicker-select.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, Input, ChangeDetectorRef } from "@angular/core";
+import { Select } from "../select/select.component";
+
+@Component({
+	selector: "ibm-timepicker-select",
+	template: `
+		<div
+			[ngClass]="{
+				'bx--select--light': theme === 'light',
+				'bx--skeleton': skeleton
+			}"
+			class="bx--select bx--time-picker__select">
+			<label *ngIf="!skeleton" [attr.for]="id" class="bx--label bx--visually-hidden">{{label}}</label>
+			<select
+				#select
+				[attr.id]="id"
+				[disabled]="disabled"
+				(change)="onChange($event)"
+				class="bx--select-input">
+				<ng-content></ng-content>
+			</select>
+			<svg *ngIf="!skeleton" class="bx--select__arrow" width="10" height="5" viewBox="0 0 10 5">
+			<path d="M0 0l5 4.998L10 0z" fill-rule="evenodd" />
+			</svg>
+		</div>
+	`,
+	styles: ["select { -webkit-appearance : none ; }"]
```

Comment:
This will be handled by the storybook, or the consuming apps code - we don't need to provide it
```suggestion
```